client/lib/mention.js

```diff
@@ -0,0 +1,67 @@
+var hueHash = require('./hue-hash')
+
+/**
+ * Determines if name contains the characters in the partial name, in order.
+ */
+module.exports.containsSubseq = function(name, part) {
+  var offset = 0
+  for(var partOffset = 0; partOffset < part.length; partOffset++) {
+    var nextChar = part.substr(partOffset, 1)
```

Comment:
No good reason, no! It's a leftover from editing.

I don't think the code will be particularly hot -- maybe with hundreds of names, it could be? But why risk it, slicing is more idiomatic too.